packages/k6-tdk/src/client/search.ts

```diff
@@ -16,56 +15,66 @@ export class Search {
     this.#api = api;
   }
 
+  sharee(search: string, itemType: ItemType): RefinedResponse<'text'> {
```

Comment:
just a suggestion: maybe it make sence to create separate `sharee.ts` file for method `sharee` and rename `sharee` as `get`. 
then we get `userClient.sharee.get(adminCredential.login, ItemType.folder)`

Just `search' I mean using REPORT request

Otherwise ok. I ran the tests - everything works fine